packages/server/graphql/types/RetrospectiveMeetingSettings.ts

```diff
@@ -69,7 +73,7 @@ const RetrospectiveMeetingSettings = new GraphQLObjectType<any, GQLContext>({
         const {orgId} = team
         const templates = await dataLoader.get('meetingTemplatesByOrgId').load(orgId)
         const organizationTemplates = templates.filter(
-          (template) =>
+          (template: MeetingTemplate) =>
```

Comment:
this is another odd one! it should already be a MeetingTemplate without the cast, right?

Reply:
Same as above - it knows the type but still complains!